# メッセージを含む例外オブジェクト

err = ArgumentError.new("引数が不正です")
raise err
